nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -125,10 +125,11 @@ def test_plot_glass_brain():
     img = _generate_img()
 
     # test plot_glass_brain with colorbar
-    plot_glass_brain(img, colorbar=True)
+    plot_glass_brain(img, colorbar=True, resampling_interpolation='nearest')
 
     # test plot_glass_brain with negative values
-    plot_glass_brain(img, colorbar=True, plot_abs=False)
+    plot_glass_brain(img, colorbar=True, plot_abs=False,
+                     resampling_interpolation='nearest')
```

Comment:
This happens implicitly up on line 105 (loop through plotting functions), okay to just have that one or would you prefer an explicit one here?